public/openapi.yml

```diff
@@ -80,6 +80,56 @@ paths:
             application/json:
               schema:
                 "$ref": "#/components/schemas/Status"
+  "/api/allocation/{nomsNumber}/primary_pom":
```

Comment:
Have you tested out the OpenAPI interfaace? http://localhost:3000/api-docs/index.html - remember to change the `defaultHost` to localhost:3000 and protocol to `http`